cmd/ipfs/daemon.go

```diff
@@ -274,6 +274,7 @@ func daemonFunc(req cmds.Request, res cmds.Response) {
 		}
 	}
 
+	fmt.Printf("Daemon is ready\n")
```

Comment:
Will this be the last thing printed even in light of scheduler fun, or slow-to-start HTTP servers and so on?